docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,175 @@
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/getting-started-guides/dcos.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Getting started with Kubernetes on DCOS
+----------------------------------------
+
+**Table of Contents**
+
+- [About Kubernetes on DCOS](#about-kubernetes-on-dcos)
+- [Prerequisites](#prerequisites)
+- [Install Kubernetes](#install-kubernetes)
+- [Operations](#operations)
+- [Uninstall Kubernetes](#uninstall-kubernetes)
+
+
+## About Kubernetes on DCOS
+
+The [Datacenter Operating System (DCOS)](https://docs.mesosphere.com/) is a collection of tools for managing distributed systems deployed on a cluster of machines (virtual or bare metal).
+
+Since Kubernetes can be configured to schedule through Mesos, the cluster kernel at the heart of DCOS, it's a natural fit to deploy Kubernetes on DCOS.
+
+The recommended way to install Kubernetes on DCOS is to use the latest pre-packaged releases in the [Mesosphere Multiverse](https://github.com/mesosphere/multiverse) DCOS package repository. This package contains a [Marathon](https://mesosphere.github.io/marathon/) deployment manifest that specifies how to run the Kubernetes core components in containers deployed and monitored by Marathon, the DCOS init system.
+
+If you want to build your own release, the package source can be found in the [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos) github repository.
+
+This tutorial will walk you through setting up Kubernetes on a DCOS cluster and starting your first pod with an nginx webserver.
+
+**NOTE:** There are [known issues with the current implementation](../../contrib/mesos/docs/issues.md).
+If you have problems completing the steps below, please [file an issue against the kubernetes-mesos project](https://github.com/mesosphere/kubernetes-mesos/issues).
+
+Further information about Kubernetes on Mesos is available in the [contrib directory](../../contrib/mesos/README.md).
+
+
+## Prerequisites
+
+- A running [DCOS cluster](https://mesosphere.com/product/)
+  - [DCOS Community Edition](https://docs.mesosphere.com/install/) is currently available on [AWS](https://mesosphere.com/amazon/), [GCE](https://mesosphere.com/google/) (early access) & [Azure](https://mesosphere.com/azure/) (coming soon).
```

Comment:
They aren't actually usable from those links. They just says to contact sales...  Do we have a better entrypoint?  I got those links from https://mesosphere.com/product/